src/utils/analysis-utils.ts

```diff
@@ -462,7 +481,9 @@ export function getAnalysisTableColumns(
   ];
 }
 
-export function downloadCSVFromTableData(analysisResult: BaselineLayerResult) {
+export function downloadCSVFromTableData(
+  analysisResult: BaselineLayerResult | PolygonAnalysisResult,
```

Comment:
This union is done quite often, consider making a type? This will allow us to search usages of this union better, and also better document why the union exists now.

Reply:
This was addressed with https://github.com/WFP-VAM/prism-frontend/pull/431/commits/8868d382d2bbf18c0a090b1fe49505d7420c8ae3.  Please resolve if you agree.  Thank you.